### PR TITLE
update LPUSHX and RPUSHX for variable elements support.

### PR DESCRIPTION
**Pull request by patpatbear**
This PR corresponds to issue #1154.

### CODE REVIEW

File: commands.json [hunk 1]
```diff
@@ -1517,7 +1517,7 @@
     "group": "list"
   },
   "LPUSHX": {
-    "summary": "Prepend a value to a list, only if the list exists",
+    "summary": "Prepend one or multiple values to a list, only if the list exists",
     "complexity": "O(1)",
```
Comment (itamarhaber) [inline commands.json]: Should be "O(N) where N is the number of elements to be added by the operation"

File: commands.json [hunk 2]
```diff
@@ -2133,7 +2134,7 @@
     "group": "list"
   },
   "RPUSHX": {
-    "summary": "Append a value to a list, only if the list exists",
+    "summary": "Append one or multiple values to a list, only if the list exists",
     "complexity": "O(1)",
```
Comment (itamarhaber) [inline commands.json]: Should be "O(N) where N is the number of elements to be added by the operation"

File: commands.json (hunk 1 above)
Comment (patpatbear) [reply commands.json]: updated complexity and history in commit 9a6afdd6f9

File: commands.json (hunk 2 above)
Comment (patpatbear) [reply commands.json]: updated complexity and history in commit 9a6afdd

**Comment by itamarhaber** [comment]
Thanks - merging after rebase and a little tweak.

**Comment by itamarhaber** [comment]
Fixes #1154

**Pull request merged**